libs/gatekeeper/config.jsonnet

```diff
@@ -0,0 +1,23 @@
+local config = import 'jsonnet/config.jsonnet';
+
+local versions = [
+  { output: '3.12', version: '3.12.0' },
+];
+
+config.new(
+  name='gatekeeper',
+  specs=[
+    {
+      local url = 'https://raw.githubusercontent.com/open-policy-agent/gatekeeper/v%s/charts/gatekeeper/crds' % v.version,
+      output: v.output,
+      prefix: '^sh\\.gatekeeper\\..*',
+      crds: [
+        '%s/external-secrets.io_clustersecretstores.yaml' % url,
+        '%s/external-secrets.io_externalsecrets.yaml' % url,
+        '%s/external-secrets.io_secretstores.yaml' % url,
```

Comment:
These CRDs seem to be related to external-secrets?

Reply:
Indeed, didn't read 😆 . And the lib is empty: https://github.com/jsonnet-libs/gatekeeper-libsonnet/tree/main/3.12